compiler/erg_parser/parse.rs

```diff
@@ -30,10 +30,11 @@ use TokenKind::*;
 macro_rules! debug_call_info {
     ($self: ident) => {
         $self.level += 1;
+        let space= " ".repeat($self.level);
         log!(
             c GREEN,
             "\n{} ({}) entered {}, cur: {}",
-            " ".repeat($self.level),
+            space.replace("    ", "･"),
```

Comment:
I think `"･".repeat(($self.level as f32 / 4.0).floor() as usize)` is more elegant and efficient.